rsmtool/notebooks/features_by_group.ipynb

```diff
@@ -38,7 +38,7 @@
     "    df_train_combined.reset_index(drop=True, inplace=True)\n",
     "\n",
     "    # decide on the the height per plot\n",
-    "    height_per_plot = 2 if len(features_used) > 15 else 4\n",
```

Comment:
We are not longer applying this condition, right? (I don't think it's part of the function either). Does it still look fine with 50 features?